docker/demo/sparksql-incremental.commands

```diff
@@ -27,8 +27,8 @@ import org.apache.hadoop.fs.FileSystem;
 val fs = FileSystem.get(spark.sparkContext.hadoopConfiguration)
 val beginInstantTime = HoodieDataSourceHelpers.listCommitsSince(fs, "/user/hive/warehouse/stock_ticks_cow", "00000").get(0)
 val hoodieIncQueryDF =  spark.read.format("org.apache.hudi").
-                      option(DataSourceReadOptions.QUERY_TYPE_OPT_KEY.key(), DataSourceReadOptions.QUERY_TYPE_INCREMENTAL_OPT_VAL).
-                      option(DataSourceReadOptions.BEGIN_INSTANTTIME_OPT_KEY.key(), beginInstantTime).
+                      option(DataSourceReadOptions.QUERY_TYPE_OPT.key(), DataSourceReadOptions.QUERY_TYPE_INCREMENTAL_OPT_VAL).
```

Comment:
@zhedoubushishi how about we drop the `_OPT` as well. Just `DataSourceReadOptions.QUERY_TYPE` ? Sounds lot shorter?

Reply:
Agree. This looks more clean. So what about other configs like xxx_PROP? shall we also remove ```_PROP```?

Reply:
I am okay keeping `_PROP` or removing. It's not as bad as `_OPT_KEY`.  It also has higher blast radius. may be leave it alone?